pop_tools/xgcm_util.py

```diff
@@ -201,5 +201,11 @@ def to_xgcm_grid_dataset(ds, **kwargs):
             """to_xgcm_grid_dataset() function requires the `xgcm` package. \nYou can install it via PyPI or Conda"""
         )
     ds_new = relabel_pop_dims(ds)
+    if 'nlat' in ds_new.dims:
+        ds_new = ds_new.drop('nlat')
+    if 'nlon' in ds_new.dims:
+        ds_new = ds_new.drop('nlon')
+    if 'z_w_top' and 'z_w' in ds_new.dims:
+        ds_new = ds_new.drop('z_w_top').rename({'z_w': 'z_w_top'}) 
```

Comment:
Would moving these lines into `relabel_pop_dims()` function (right before the return statement) cause any problem?

https://github.com/NCAR/pop-tools/blob/b76f5c1e5536ad592a825743a2d27bad2f485b54/pop_tools/xgcm_util.py#L100-L102

Reply:
I think that should work as well.

Reply:
Does this work? 

If `relabel_pop_dims` does its job, `nlat` and `nlon` should not be in `ds_new.dims` but will be in `ds_new.coords`. In that case you should use `.drop_vars` (does the same thing but is clearer about what's being dropped)

Reply:
so 
if 'nlat' in ds_new.coords:
    ds_new = ds_new.drop_vars('nlat')

and so on?

Reply:
I think so.

We should add a test with `assert 'nlon' not in ds_new.dims` and so on.

Reply:
I changed ```drop``` to ```drop_vars``` and added a line with ```assert```, not sure if that is what you had in mind though.

Reply:
Almost. Let's put the `assert` statement in this test:
https://github.com/NCAR/pop-tools/blob/b76f5c1e5536ad592a825743a2d27bad2f485b54/tests/test_xgcm_util.py#L14-L23

We should also `assert z_w not in ds_new`

You can run the tests by (after conda installing `pytest`)
```
cd pop-tools/tests
pytest
```